404.html

```diff
@@ -1,6 +1,10 @@
 ---
 layout: default
 ---
+<head>
+  <meta charset="UTF-8">
+</head>
+<body style="background-color:powderblue;">
```

Comment:
This color is inappropriate. you can see the website by executing the following command:

```
$ bundle exec jekyll serve
```